manifests/e/ente-io/auth-desktop/3.0.4/ente-io.auth-desktop.installer.yaml

```diff
@@ -0,0 +1,26 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: ente-io.auth-desktop
+PackageVersion: 3.0.4
+
+InstallerType: inno
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches: 
+  Silent: /VERYSILENT /SUPPRESSMSGBOXES
+  SilentWithProgress: /SILENT /SUPPRESSMSGBOXES
+Scope: machine
+ElevationRequirement: elevatesSelf
+UpgradeBehavior: install
+ReleaseDate: 2024-05-21
+Installers:

```

Comment:
```suggestion
Dependencies:
  PackageDependencies:
     - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

Reply:
Is there a way to suggest a package instead of adding as a dependency?

Reply:
Suggest? In which way - like an optional dependency? The package manager doesn't currently support this functionality, but feel free to add a [feature request](https://github.com/microsoft/winget-cli/issues/new?assignees=&labels=Issue-Feature&projects=&template=Feature_Request.yml).

Reply:
> Suggest? In which way - like an optional dependency? The package manager doesn't currently support this functionality, but feel free to add a [feature request](https://github.com/microsoft/winget-cli/issues/new?assignees=&labels=Issue-Feature&projects=&template=Feature_Request.yml).

You mean like this one?
* https://github.com/microsoft/winget-cli/issues/3928